web/app.fluidity.money/contexts/EthereumProvider.tsx

```diff
@@ -406,15 +456,16 @@ const EthereumFacade = ({
     usdcAmt: BN,
     fusdcAmt: BN,
     wethAmt: BN,
-    slippage: BN
-  ): Promise<StakingDepositsRes | undefined> => {
+    slippage: BN,
+    maxTimestamp: BN,
+  ): Promise<TransactionResponse | undefined> => {
```

Comment:
Is there any way to not return undefined here, instead returning some type that contains the response that could be checked instead, or a tuple?